docs/spec/v1beta2/helmrepositories.md

```diff
@@ -418,6 +521,10 @@ and reports `Reconciling` and `Stalled` conditions where applicable to
 provide better (timeout) support to solutions polling the HelmRepository to become
 `Ready`.
 
+The `artifact` and `storage` related condition types are only supported by
+HTTP/s Helm repositories. OCI Helm repositories do not use storage, they do not have
+any artifacts.
```

Comment:
I would instead maybe simply state that the OCI Helm repositories only make use of Reconciling, Ready and Stalled. Which is easier to comprehend than ruling out what it doesn't.